frontend/src/views/Monitor.vue

```diff
@@ -30,6 +30,19 @@ export default {
 }
 </script>
 
+<style lang="scss">
```

Comment:
This commit should be squashed into the first commit.

Reply:
👍